src/info/repo/project.rs

```diff
@@ -12,8 +13,8 @@ pub struct ProjectInfo {
 }
 
 impl ProjectInfo {
-    pub fn new(repo: &Repository, repo_url: &str) -> Result<Self> {
-        let repo_name = get_repo_name(repo_url)?.unwrap_or_default();
+    pub fn new(repo: &Repository, repo_url: &str, manifest: &Option<Manifest>) -> Result<Self> {
```

Comment:
`&Option<T>` should be avoided in favour of `Option<&Manifest>`, which is way more usable. The caller owning `Option<T>` can do `my_option.as_ref()` to pass it along.

Reply:
done, thanks